easybuild/easyconfigs/b/Beast/Beast-2.7.3-GCC-11.3.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'Tarball'
+
+name = 'Beast'
+version = '2.7.3'
+
+homepage = 'https://beast2.org'
+description = """ BEAST is a cross-platform program for Bayesian MCMC analysis of molecular 
+ sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using 
+ strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies 
+ but is also a framework for testing evolutionary hypotheses without conditioning on a single 
+ tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted 
+ proportional to its posterior probability. """
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+
+_archs = {'x86_64': 'x86', 'aarch64': 'aarch64'}
+
+source_urls = ['https://github.com/CompEvol/beast2/releases/download/v%(version)s/']
+sources = ['BEAST.v%%(version)s.Linux.%s.tgz' % _archs[ARCH]]
+checksums = ['234af8572e827c1c856f0e1ad46e61fa3b3b875c6b5d246f67839ec7c1bf40b5']
```

Comment:
surely the different `_archs` will have different checksums?

Reply:
indeed, good catch